integrations/lib/multiwoven/integrations/destination/http/client.rb

```diff
@@ -0,0 +1,114 @@
+# frozen_string_literal: true
+
+require "base64"
```

Comment:
Can we add this in `integrations/lib/multiwoven/integrations.rb` this will help others also use it in integrations when need arises as it's a common library

Reply:
removing this as it is not needed anymore